.github/workflows/test.yml

```diff
@@ -1,11 +1,13 @@
-name: test
+name: Test Discombobulator
 
 on:
```

Comment:
add `on: push` then the branch in questionto this. This will run the same workflow as on pr

Reply:
I don't want the workflow to run on every push. It runs when you open a PR (even just draft) to develop which will then again run it for every commit as well

Reply:
Alright?